### PR TITLE
Remove verbose text during building

### PR DESCRIPTION
**Pull request by andy31415**
We need non-fatal diagnostics to not be printed by default during builds, since that just makes logs noisy.

Deleting a message about input matter IDL detection.

### CODE REVIEW

**Review by mspang — APPROVED** [review]

**Review by yufengwangca — APPROVED** [review]

File: src/controller/tests/data_model/TestRead.cpp [hunk 1]
```diff
@@ -1769,10 +1769,31 @@ void TestReadInteraction::TestReadHandler_MultipleSubscriptions(nlTestSuite * ap
...
+        params.mOnSubscriptionEstablishedCb = onSubscriptionEstablishedCb;
+        params.mOnResubscriptionAttemptCb   = nullptr;
+        params.mOnDoneCb                    = nullptr;
+        params.mMinIntervalFloorSeconds     = 0;
+        params.mMaxIntervalCeilingSeconds   = 20;
+        params.mReportType                  = app::ReadClient::InteractionType::Subscribe;
+        params.mKeepSubscriptions           = true;
+        params.mIsFabricFiltered            = false;
+
+        // Increase timeout, so that slow versions (specifically gcc_debug) does not time out.
```
Comment (bzbarsky-apple) [inline src/controller/tests/data_model/TestRead.cpp:1786]: Why is this change needed?  Why are things fine without this change before but not fine after?

File: src/controller/tests/data_model/TestRead.cpp [hunk 2]
```diff
@@ -1769,10 +1769,31 @@ void TestReadInteraction::TestReadHandler_MultipleSubscriptions(nlTestSuite * ap
...
+        params.mTimeout = chip::System::Clock::Seconds16(15);
+
+        CHIP_ERROR err =
+            Controller::detail::ReportAttribute<Clusters::UnitTesting::Attributes::ListStructOctetString::TypeInfo::DecodableType>(
```
Comment (bzbarsky-apple) [inline src/controller/tests/data_model/TestRead.cpp:1790]: You should not be using "detail" APIs here.  There are public APIs for all this; please use them.

File: src/credentials/tests/BUILD.gn [hunk 3]
```diff
@@ -46,14 +47,20 @@ chip_test_suite("tests") {
...
     "TestFabricTable.cpp",
     "TestGroupDataProvider.cpp",
     "TestPersistentStorageOpCertStore.cpp",
   ]
 
+  if (!chip_config_memory_debug_dmalloc) {
+    # We have many DUT vectors and dmalloc requires a lot of time to validate
+    # heap operations for large crypto processing. Skip this test for
+    # dmalloc.
+    test_sources += [ "TestCommissionerDUTVectors.cpp" ]
```
Comment (bzbarsky-apple) [inline src/credentials/tests/BUILD.gn:61]: Why is this change needed for a _logging_ change?

File: src/transport/tests/TestSessionManager.cpp [hunk 4]
```diff
@@ -765,10 +765,19 @@ void SessionAllocationTest(nlTestSuite * inSuite, void * inContext)
         handle.Value()->AsSecureSession()->MarkForEviction();
     }
 
+#if CHIP_CONFIG_MEMORY_DEBUG_DMALLOC
```
Comment (bzbarsky-apple) [inline src/transport/tests/TestSessionManager.cpp:768]: Why is this needed for a logging change?

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

**Pull request closed without merging**